forms/EmailField.php

```diff
@@ -1,54 +1,59 @@
 <?php
+
 /**
- * Text input field with validation for correct email format
- * according to RFC 2822.
- * 
+ * Text input field with validation for correct email format according to RFC 2822.
+ *
  * @package forms
  * @subpackage fields-formattedinput
  */
 class EmailField extends TextField {
-
+	/**
+	 * {@inheritdoc}
```

Comment:
I don't agree. Knowing there's PHPDoc info on a parent is better than not knowing. It also creates a habit of strictly opt-out PHPDocs.